openedx/adg/lms/branding_extension/helpers.py

```diff
@@ -45,3 +46,18 @@ def get_copyright():
         year=datetime.today().year,
         org_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
     )
+
+
+def is_referred_by_login_or_register(request):
+    """
+    Returns True if user is redirected to root from login or register, otherwise False
```

Comment:
Please update docstring, as this method has nothing to do with root. It only checks that request is redirected from login or register flow.